configs/base.js

```diff
@@ -1,11 +1,11 @@
 const { fixupPluginRules } = require("@eslint/compat");
 const js = require("@eslint/js");
+const eslintConfigPrettier = require("eslint-config-prettier/flat");
```

Comment:
> Manually include prettier eslint config to disable eslint rules that conflict with prettier


I don't quite get the positioning here -- it seems like the intent is that this plugin no longer assumes use of prettier, but here you still expect it to be used with prettier? It seems weird IMO to keep this in the plugin when you are removing prettier from the install instructions -- specifically because someone who sets up a new project and does **not** include prettier in their project is still getting eslint-config-prettier which will (now unnecessarily) disable some other eslint rules.